src/govuk/components/summary-list/summary-list.yaml

```diff
@@ -156,10 +156,10 @@ examples:
             items:
               - href: '#'
                 html: Golygu<span class="govuk-visually-hidden"> enw</span>
-                visuallyHiddenText: false
+                visuallyHiddenText:
```

Comment:
Unlike the error message component, there's no default for the visually hidden text for each action:

https://github.com/alphagov/govuk-frontend/blob/dc6fc67d4dca1574194b8337ff0e3cfa07b14dd1/src/govuk/components/summary-list/template.njk#L4-L6

So we _could_ just omit this option entirely, and it'd have the same effect.